components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/firewallchecktrigger/firewallchecktrigger_test.go

```diff
@@ -83,6 +83,22 @@ const (
 			  "msg": "",
 			  "check": "",
 			  "checks": [
+				{
+					"title": "Check for reachability of service at destination port from the source node",
+					"passed": true,
+					"success_msg": "The <protocol> service running at <destination_node_ip>:<destination_node_port> is reachable fro <source_node_ip>",
```

Comment:
typo - `is reachable fro` 
This is should get caught on unit testing, Can you please check and fix it if we are not asserting success_msg?

Reply:
This didn't get caught, because expected result also had this typo.